nixos/modules/virtualisation/virtualbox-image.nix

```diff
@@ -45,6 +45,35 @@ in {
           The file name of the VirtualBox appliance.
         '';
       };
+      extraDisk = mkOption {
+        description = ''
+          Optional extra disk/hdd configuration.
+          The disk will be an 'ext4' partition on a separate VMDK file.
+        '';
+        default = null;
+        example = {
+          label = "storage";
+          mountPoint = "/home/demo/storage";
+          size = 100 * 1024;
+        };
+        type = types.nullOr (types.submodule {
+          options = {
+            size = mkOption {
```

Comment:
I like if the option name expresses the unit when it's not obvious, so personally I'd name this option `size_MiB`. Otherwise when one sees configuration setting this it is necessary to consult the docs to know.

Reply:
That battle is already lost unfortunately - there's `(baseImage,memory)Size` (both in MiB) too, and I guess, we should stay consistent with these options.

Reply:
> That battle is already lost unfortunately - there's `(baseImage,memory)Size` (both in MiB) too, and I guess, we should stay consistent with these options.

Right then.